src/resources/swagger.yaml

```diff
@@ -842,6 +842,28 @@ paths:
         '200':
           description: 'returns new entity exception after deletion'
 
+  /exception/missing-entity/${programId}:
+    post:
+      tags:
+        - Exception
+      summary: Add submitter donor ids for exception
+      parameters:
+        - $ref: '#/components/parameters/PathProgramId'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MissingEntityExceptionDonorIds'
+
+      responses:
+        '201':
```

Comment:
This is Create or Update so it may not actually create an entity. Let's just use Status 200